schema/artist/highlights.js

```diff
@@ -1,23 +1,51 @@
 import { GraphQLBoolean, GraphQLObjectType, GraphQLList, GraphQLString } from "graphql"
```

Comment:
If all of the fields for market data are just going to be root on the artist, we could just remove this file completely.

Or, we can duplicate a bit: allow partners to exist on artist, but also have this (maybe more specialized/filtered) call here.